Sources/OpenAI/Public/Models/AudioSpeechResult.swift

```diff
@@ -25,7 +25,7 @@ public struct AudioSpeechResult {
     public func saveAs(_ name: String, format: AudioSpeechQuery.AudioSpeechResponseFormat, to path: URL) throws {
         guard let data = audioData else {
             throw NSError(
```

Comment:
What do you think about using Swift Error instead of Objective-C NSError. Do we have some limitations?